src/js/jsx/search/SearchBar.jsx

```diff
@@ -543,6 +559,7 @@ define(function (require, exports, module) {
                         className="dialog-search-bar"
                         options={this.state.options}
                         startFocused={true}
+                        filters={filter}
```

Comment:
It's weird that the singular word `filter` is being used to indicate a list of filters here?